pootle/apps/pootle_store/views.py

```diff
@@ -522,8 +527,8 @@ def get_context_data(self, *args, **kwargs):
             'suggestions': suggestions,
             'suggestions_dict': {x.id: dict(id=x.id, target=x.target.strings)
                                  for x in suggestions},
-            "critical_checks": list(
-                self.object.get_critical_qualitychecks()),
+            "failing_checks": failing_checks,
```

Comment:
This key doesn't reflect it if failing **critical**.

Reply:
im not sure it matters - the point is its failing hard enough that it blocks